html-aam/index.html

```diff
@@ -16147,6 +16152,26 @@ <h4>`fieldset` Element Accessible Name Computation</h4>
             <li>Otherwise, there is no <a data-cite="accname-1.2/#dfn-accessible-name">accessible name</a>.</li>
           </ol>
         </section>
+        <section>
+          <h4>`optgroup` Element Accessible Name Computation</h4>
+          <ol>
+            <li>
+              If the `optgroup` element has an <a data-cite="wai-aria-1.2/#aria-label">`aria-label`</a> or an <a data-cite="wai-aria-1.2/#aria-labelledby">`aria-labelledby`</a> attribute the
+              <a data-cite="accname-1.2/#dfn-accessible-name">accessible name</a> is to be calculated using the algorithm defined in
```

Comment:
label should come before legend
if label is used with legend, then legend should become description for optgroup